go/vt/vttablet/tabletserver/tabletserver.go

```diff
@@ -2024,7 +2024,10 @@ func (tsv *TabletServer) StreamPoolSize() int {
 
 // SetTxPoolSize changes the tx pool size to the specified value.
 func (tsv *TabletServer) SetTxPoolSize(ctx context.Context, val int) error {
-	return tsv.te.txPool.scp.conns.SetCapacity(ctx, int64(val))
+	if err := tsv.te.txPool.scp.conns.SetCapacity(ctx, int64(val)); err != nil {
```

Comment:
It's worth writing a comment here as to why two pools have to be set, not just one.